src/projects/projects-page.html

```diff
@@ -62,14 +67,17 @@
           </a>
         </template>
       </preview-list>
-      <project-pull-requests hidden$="[[!_shouldShowProjects(name, pr)]]" route="{{nameTail}}" tabindex="0" owner="[[owner.owner]]" name="[[name.name]]"></project-pull-requests>
+      <project-pull-requests hidden$="[[!_shouldShowProjects(projectTitle, pr)]]" route="{{nameTail}}" tabindex="0"
```

Comment:
This method does not seem to work.
_shouldShowProjects returned true, but it's still shown.
I tried debugging, but could not find the cause.